src/reducers/entities/users.js

```diff
@@ -268,6 +268,7 @@ function profilesInChannel(state = {}, action) {
             user_id: action.data.user_id,
         }});
 
+    case ChannelTypes.SELECT_CHANNEL:
```

Comment:
Can you please explain this? This really makes me nervous

Reply:
From our discussion, I've created a new action to replace the user list when getting a new one in the state instead of adding both